setup.py

```diff
@@ -10,6 +10,6 @@
       package_data={'pypif_sdk' : ['func/elements.json']},
       install_requires=[
           'pypif>=2.0.0,<4',
-          'citrination_client>=3,<5',
+          'citrination_client>=3',
```

Comment:
```suggestion
          'citrination_client>=3',
```
```suggestion
          'citrination_client>=3,<7',
```
We should always have an upper bound on these version numbers.

Reply:
Oh for sure